app/controllers/api/motorcycles_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::MotorcyclesController < ApplicationController
+  before_action :set_user, only: [:create]
+
+  def index
+    @motors = Motorcycle.all
+    render json: @motors
+  end
```

Comment:
Good job on implanting  the controller, but it would be better if you handle errors,
For example : 
```
if @motors
      render json: @motors
    else
      render json: { error: 'Opps! No motor yet!' }, status: :not_found
    end
```

Reply:
Hello, @sentayhu19. Thank you for the suggestion, however, I don't think there will be any issues regarding catching errors since `@motors `will always return an empty array `[ ]` if nothing is found. Thank you.

Reply:
okay but am not saying there will be an issue, it also works as you said with  `[ ]`. this one is to make our backend respond to those kinds of requests with `404`.